prow/gerrit/adapter/adapter_test.go

```diff
@@ -757,5 +816,10 @@ func TestProcessChange(t *testing.T) {
 				t.Errorf("tc %s - BaseSHA should be abc, got %s", tc.name, prowjobs[0].Spec.Refs.BaseSHA)
 			}
 		}
+		if tc.shouldSkipReport {
+			if gc.reviews > 0 {
+				t.Errorf("tc %s - expected no comments, got: %d", tc.name, gc.reviews)
```

Comment:
Ideally we refactor this unit test to match modern practices, aka

```golang
// deprecated
for _, tc := range testCases {
  // whatever
  tc.Errorf("tc %s ...", tc.name)
}

// expected
for _, tc := range testCases {
  t.Run(tc.name, func(t *testing.T) {
    t.Error("...") // name automatically appended
  }
}
```

This makes for better unit tests: cases run in parallel which make them faster as well as more likely to be hermetic.

Typically I convert test functions that still use the deprecated pattern when I'm modifying them (feel free to pass if you don't feel up to it though).

Reply:
FYI I think you also need to call [`t.Parallel()`](https://golang.org/pkg/testing/#T.Parallel) at the beginning of the anonymous run function if you want the test cases to run in parallel. ref: https://golang.org/pkg/testing/#hdr-Subtests_and_Sub_benchmarks

Reply:
Thank you for the reviews. Fixed.